app/routes/companyInterest/utils.js

```diff
@@ -26,18 +26,61 @@ export const SemesterNavigation = ({ title }: { title: Node }) => (
 );
 
 export const SEMESTER_TRANSLATION = {
-  spring: 'Vår',
-  autumn: 'Høst'
+  spring: {
+    norwegian: 'Vår',
+    english: 'Spring'
+  },
+  autumn: {
+    norwegian: 'Høst',
+    english: 'Autumn'
+  }
 };
 
 export const semesterToText = ({
   semester,
-  year
+  year,
+  language
 }: {
   semester: string,
-  year: number | string
+  year: number | string,
+  language: string
 }) => {
   return semester === 'spring' || semester === 'autumn'
-    ? `${SEMESTER_TRANSLATION[semester]} ${year}`
+    ? `${SEMESTER_TRANSLATION[semester][language]} ${year}`
     : '';
 };
+
+export const interestText = {
+  comment: {
+    norwegian:
+      'For at vi skal kunne legge tilrette for deres ønsker på best ' +
+      'mulig måte, ønsker vi at dere skriver litt om hvordan dere ' +
+      'ønsker å gjennomføre arrangementet.',
+    english:
+      'Please explain your expectations for the event to help us take your preferences ' +
+      'into consideration.'
+  },
+  text: {
+    first: {
+      norwegian:
+        'Dersom dere ønsker noe utenfor de vanlige rammene, huk gjerne av på ' +
+        '“Alternativt arrangement” og skriv en kommentar om hva dere kunne ' +
+        'tenkt dere å gjøre i kommentarfeltet. Vi i Abakus ønsker å kunne tilby ' +
+        'et bredt spekter av arrangementer som er gunstig for våre studenter. ',
+      english:
+        'If you want an event out of the ordinary, you can check the “other” box and ' +
+        'write a comment explaining what you have in mind. In Abakus, we want to ' +
+        'offer a wide range of events that are beneficial for our students. '
+    },
+    second: {
+      norwegian:
+        'Kommentarfeltet kan også brukes til å spesifisere annen informasjon, ' +
+        'som for eksempel hvilken teknologi dere ønsker å lære bort hvis dere ' +
+        'får faglig arrangement.',
+      english:
+        'The comment box can also be used for other types of information, for example' +
+        'what type(s) of technology you would like to show our students if you are ' +
+        'holding an academic event.'
```

Comment:
`such as which technologies you would like to present for our students when hosting an academic event`